ReleaseTooling/Sources/FirebaseManifest/FirebaseManifest.swift

```diff
@@ -23,9 +23,9 @@ import Foundation
 public let shared = Manifest(
   version: "8.0.0",
   pods: [
-    Pod("FirebaseCoreDiagnostics"),
-    Pod("FirebaseCore"),
-    Pod("FirebaseInstallations"),
+    Pod("FirebaseCoreDiagnostics", zip: true),
```

Comment:
Otherwise we always build these three from a published podspec and never from head.